### ISSUE TITLE
Logs are double printing - Logs are double printing

### ISSUE THREAD

**Issue by jkrauska** [CodaProtocol/coda#2993]
probably related to log rotate

testnet nodes:

```
{"timestamp":"2019-07-24 03:01:19.294618Z","level":"Trace","source":{"module":"Coda","location":"File \"app/cli/src/coda.ml\", line 291, characters 65-72"},"message":"Error reading daemon.json: $error","metadata":{"error":"(Unix.Unix_error\"No such file or directory\"open\"((filename /home/admin/test-coda/daemon.json) (mode (O_RDONLY O_CLOEXEC)) (perm 0o0))\")","pid":12334}}

{"timestamp":"2019-07-24 03:01:19.294618Z","level":"Trace","source":{"module":"Coda","location":"File \"app/cli/src/coda.ml\", line 291, characters 65-72"},"message":"Error reading daemon.json: $error","metadata":{"error":"(Unix.Unix_error\"No such file or directory\"open\"((filename /home/admin/test-coda/daemon.json) (mode (O_RDONLY O_CLOEXEC)) (perm 0o0))\")","pid":12334}}
```

**Comment by imeckler** [comment]
@cmr and @nholland94 believe they fixed this. It needs to be tested.

**Comment by jkrauska** [comment]
@Schmavery @nholland94 @cmr 

still happening in beta 0.0.2 -- can you point to PR where we think this was fixed?

**Comment by imeckler** [comment]
@Schmavery to investigate

**Comment by Schmavery** [comment]
I can't repro this on master running `coda daemon`. Can anyone else?

**Comment by jkrauska** [comment]
here's how we call the daemon on production testnets

```
coda daemon -background -tracing -no-bans -log-json -log-level Trace -client-port 8301 -external-port 8302 -config-directory /home/admin/test-coda -external-ip 13.56.250.68 -rest-port 8304 -peer 13.56.250.68:8303 -propose-key /home/admin/wallet-keys/proposerkey -unsafe-track-propose-key```

perhaps one of those options triggers it?

**Comment by Schmavery** [comment]
To make this issue a little easier to follow, the error seems to be as follows:
coda.log has double-printed logs when the daemon with `-background -log-json`

**Comment by Schmavery** [comment]
Ok this should be fixed by #3243
